ngraph/core/reference/include/ngraph/runtime/reference/utils/span.hpp

```diff
@@ -92,6 +92,34 @@ namespace ngraph
                     return {m_data + offset, std::min(size, m_size - offset)};
                 }
 
+                Span& drop_front(std::size_t number_of_elements)
+                {
+                    if (number_of_elements < m_size)
+                    {
+                        m_data += number_of_elements;
+                        m_size -= number_of_elements;
+                    }
+                    else
+                    {
+                        m_size = 0;
+                    }
+                    return *this;
+                }
+
+                Span& drop_back(std::size_t number_of_elements)
```

Comment:
Add comment describing the function behaviour.

Reply:
done